File: leftover.js
function leftover(m, p, q) {
    return m * (100 - p) * (100 - q) / 10000;
}

const [m, p, q] = require("fs").readFileSync("/dev/stdin", "utf8").trim().split(" ").map(Number);
console.log(leftover(m, p, q));

File: leftover.scala
import scala.io.StdIn._

object Main extends App {
    def leftover(m: Double, p: Double, q: Double): Double = {
        return m * (100 - p) * (100 - q) / 10000
    }

    val Array(m, p, q) = readLine().split(" ").map(_.toDouble)
    println(leftover(m, p, q))
}